app/assets/javascripts/views/history_view.js

```diff
@@ -5,41 +5,81 @@ module.exports = Backbone.View.extend({
   initialize: function() {
     _.bindAll(this, 'addActivities', 'connectionError');
 
-    this.currentStory = {};
-
-    this.$el.html(this.template({id: 'history', name: I18n.t('projects.show.history')}));
+    this.$el.html(this.template({
+      id: 'history',
+      className: 'activity_column',
+      name: I18n.t('projects.show.history')
+    }));
+    this.$el.addClass('history_column');
+    this.$column = this.$('#history');
     this.$loadingSpin = $('.loading-spin');
   },
 
-  addActivities: function(activities, a, b) {
-    this.$el.html('');
-    this.trigger('change:currentStory', this.currentStory);
+  events: {
+    'change:currentStory': 'handleStoryChange',
+    'click a.toggle-column': 'toggle'
+  },
+
+  toggle: function() {
+    this.$el.toggle();
+    this.stopListening(this.currentStory, 'sync');
+  },
+
+  handleStoryChange: function(event) {
+    this.setHistoryTitle(event)
+    this.fetchActivities(event)
+    event.stopPropagation();
```

Comment:
It's a better practice to call `stopPropagation` as the first statement of the method.